docs/kratos/social-signin/95_account-linking.mdx

```diff
@@ -1,52 +1,66 @@
   profile
-- to remove a social sign-in provider link from the profile (possible only when multiple sign-in methods are available to prevent
+- to remove a social login provider link from the profile (possible only when multiple login methods are available to prevent
   locking users out from accounts)
 
-## Link accounts
+### Link accounts
 
 Users can link accounts manually through their account's settings page. To try out account linking, use the Ory Account
 Experience. Follow these steps:
 
-1. Configure at least two sign-up methods in your Ory project. One of these methods must be through a social sign-in provider.
-2. Go to your project's Ory Account Experience at `https://$PROJECT_SLUG.projects.oryapis.com/ui` and sign up.
+1. Configure at least two sign-up methods in your Ory project. One of these methods must be through a social login provider.
+2. Go to your project's Ory Account Experience at `https://$PROJECT_SLUG.projects.oryapis.com/ui/registration` and sign up.
 3. After you sign up, go to **Account Settings** and navigate to the **Social Sign In** section.
-4. Select one of the buttons to link an available social sign-in provider.
+4. Select one of the buttons to link an available social login provider.
 
-## Unlink accounts
+### Unlink accounts
 
-Users with multiple sign-in methods can unlink social sign-in providers from their account through their account's settings page.
-To try out account un-linking, use the Ory Account Experience. Follow these steps:
+Users with multiple login methods can unlink social login providers from their account through their account's settings page. To
+try out account un-linking, use the Ory Account Experience. Follow these steps:
 
 1. Go to your project's Ory Account Experience at `https://$PROJECT_SLUG.projects.oryapis.com/ui` and sign in with a user account
-   with multiple sign-in methods available.
+   with multiple login methods available.
 2. Go to **Account Settings** and navigate to the **Social Sign In** section.
-3. Use the buttons to unlink a social sign-in provider.
+3. Use the buttons to unlink a social login provider.
+
+## Account linking on login and registration
 
-## Automatic account linking
+Users can link their accounts to social login providers during login or registration. This is useful when users sign up with a
+social login provider and later want to link their account to another social login provider or to an email and password account:
 
-Users can link social sign-in accounts on login without interaction using a secure flow. This is how it works:
+1. The user creates an account with the identifier `alice@example.com`, which is also a verifiable address, and a login method
+   such as a password or passkey.
+1. When signing in later, the user signs in with a SSO login provider. That SSO account (through the OIDC userinfo endpoint or the
+   ID token) returns a **verified email** of `alice@example.com`.
```

Comment:
AFAIK, the OIDC provider's email does not need to be verified.

Reply:
You are right - resolved